master/usage/assigning-ip-addresses-topology.md

```diff
@@ -0,0 +1,145 @@
+
+1. Delete the default IP pool.
+
+	Since the `default-ipv4-ippool` IP pool resource already exists and accounts for the entire `/16` block, we will have to delete this first:
+
+   ```
+   calicoctl delete ippools default-ipv4-ippool
+   ```
+
+2. Label the nodes.
+   ```
+   kubectl label nodes kube-node-0 rack=0
+   kubectl label nodes kube-node-1 rack=0
+   kubectl label nodes kube-node-2 rack=1
+   kubectl label nodes kube-node-3 rack=1 
+   ```
+3. Create an IP pool for each rack.
+
+   ```
+   calicoctl create -f -<<EOF
+   apiVersion: projectcalico.org/v3
+   kind: IPPool
+   metadata:
+     name: rack-0-ippool
+   spec:
+     cidr: 192.168.0.0/24
+     ipipMode: Always
+     natOutgoing: true
+     nodeSelector: rack == "0"
+EOF
+   ```
+
+   ```
+   calicoctl create -f -<<EOF
+   apiVersion: projectcalico.org/v3
+   kind: IPPool
+   metadata:
+     name: rack-1-ippool
+   spec:
+     cidr: 192.168.1.0/24
+     ipipMode: Always
+     natOutgoing: true
+     nodeSelector: rack == "1"
+EOF
+   ```
+
+   We should now have two enabled IP pools, which we can see when running `calicoctl get ippool -o wide`:
+
+   ```
+   NAME                  CIDR             NAT    IPIPMODE   DISABLED   SELECTOR
+   rack-1-ippool         192.168.0.0/24   true   Always     false      rack == "1"
+   rack-2-ippool         192.168.1.0/24   true   Always     false      rack == "2"
+   ```
+   {: .no-select-button}
+
+4. Verify that the IP pool node selectors are being respected.
+
+   We will create an nginx deploy with five replicas to get a workload running on each node.
+
+   ```
+   kubectl create deployment nginx --image nginx
+   kubectl scale deployment nginx --replicas 5
+   ```
+
+   Check that the new workloads now have an address in the proper IP pool allocated for rack that the node is on by running `calicoctl get wep -owide`:
+
+   ```
+   NAME                                            WORKLOAD               NODE          NETWORKS            INTERFACE         PROFILES                          NATS
+   kube--node--0-k8s-nginx--5c7588df--4g2b9-eth0   nginx-5c7588df-4g2b9   kube-node-0   192.168.0.65/32    cali79b6f790a38   kns.default,ksa.default.default
+   kube--node--1-k8s-nginx--5c7588df--sl2wq-eth0   nginx-5c7588df-sl2wq   kube-node-1   192.168.0.97/32    calic790f38759d   kns.default,ksa.default.default
+   kube--node--2-k8s-nginx--5c7588df--2bcgv-eth0   nginx-5c7588df-2bcgv   kube-node-2   192.168.1.33/32    calicd34697b75c   kns.default,ksa.default.default
+   kube--node--3-k8s-nginx--5c7588df--dj66l-eth0   nginx-5c7588df-dj66l   kube-node-3   192.168.1.129/32   cali7a3f8f86b3a   kns.default,ksa.default.default
+   kube--node--2-k8s-nginx--5c7588df--lzrhk-eth0   nginx-5c7588df-lzrhk   kube-node-2   192.168.1.34/32    calida5e7e2baf2   kns.default,ksa.default.default
+   ```
+   {: .no-seleck-button}
+
+   Note how the fourth byte of the IP address assigned to the workload differ based on what node that they been scheduled to. 
```

Comment:
I think third byte / octet was correct here - it's the `.0` vs `.1` that indicates which pool was used.

Reply:
Ah. you're referring to block.

I think this sentence is still a bit weird, because the fourth byte is the address and so differs by pods on the same host as well. Probably just worth calling out the per-rack aggregation here.